basic/src/tasks/components/TaskList.tsx

```diff
@@ -0,0 +1,61 @@
+import { deleteTasks, getTasks, useQuery } from "wasp/client/operations";
+import { Button } from "../../shared/components/Button";
+import { TaskListItem } from "./TaskListItem";
+
+export function TaskList() {
+  const { data: tasks, isLoading, isSuccess } = useQuery(getTasks);
+
+  if (isLoading) {
+    return <p>Loading...</p>;
+  }
+
+  if (!isSuccess) {
+    return <p className="text-red-500">Error loading tasks.</p>;
+  }
+
+  const completedTasks = tasks.filter((task) => task.isDone);
+
+  async function deleteCompletedTasks() {
+    if (completedTasks.length === 0) {
+      return;
+    }
+
+    const taskIds = completedTasks.map((task) => task.id);
+    try {
+      await deleteTasks(taskIds);
+    } catch (err: unknown) {
+      window.alert(`Error while deleting tasks: ${String(err)}`);
+    }
+  }
+
+  return (
+    <div className="flex flex-col gap-6">
+      <div className="flex items-center justify-between">
+        <div>
```

Comment:
```suggestion
        <div className="text-sm text-neutral-500">
```

Making it a bit more muted since it's secondary information.